Source/WebCore/dom/Element.h

```diff
@@ -901,6 +902,9 @@ class Element : public ContainerNode {
     static AtomString makeTargetBlankIfHasDanglingMarkup(const AtomString& target);
 
 private:
+#if ENABLE(ATTACHMENT_ELEMENT)
+    friend class AttachmentAssociatedElement;
```

Comment:
I don't think we should do this. I'd much rather have `serializeElementWithoutAttributesAndChildren` be publicly accessible than to make this a friend of `Element`.